docs/tutorial-basics/create-a-page.md

```diff
@@ -47,6 +48,16 @@ Add the Route to the Pages Array: After defining the new page route, add it to t
 const Pages: PageType[] = [MyNewPageRoute];
 ```
 
+## PageType
```

Comment:
Thank you very much for this update. Like I said in my previous comment, I'll merge this PR and then I'll add more info to this section.